metadata/src/test/java/org/apache/kafka/controller/ClusterControlManagerTest.java

```diff
@@ -407,13 +405,8 @@ public void testPlaceReplicas(int numUsableBrokers) throws Exception {
                 new PlacementSpec(0,
                     1,
                     (short) 3),
-                new ClusterDescriber() {
-                    @Override
-                    public Iterator<UsableBroker> usableBrokers() {
-                        return clusterControl.usableBrokers();
-                    }
-                }
-            );
+                    () -> clusterControl.usableBrokers()
+            ).assignments().stream().map(x -> x.replicas()).collect(Collectors.toList());
```

Comment:
Did you consider changing the type of `results` to `TopicAssignment` and fixing the `for` loop below?